server/audit_math/ballot_polling.py

```diff
@@ -16,20 +16,40 @@ class SampleSizeOption(TypedDict):
 
 
 def get_sample_size(
-    risk_limit: int, contest: Contest, sample_results: Dict[str, int], mathtype: str
+    risk_limit: int,
+    contest: Contest,
+    sample_results: Dict[str, Dict[str, int]],
+    mathtype: str,
```

Comment:
should the type of `mathtype` be `BallotPollingType`? also a more conventional variable name would be `math_type`

Reply:
ah, for sure.